app/assets/javascripts/app/project.js

```diff
@@ -39,7 +40,7 @@ App.addChild('Project', _.extend({
     if($tab.length > 0){
       this.onTabClick({ currentTarget: $tab });
 
-      if(($tab.prop('id') == 'project_metrics_link') || ($tab.prop('id') == 'project_reports_link') || ($tab.prop('id') == 'basics_link') || ($tab.prop('id') == 'dashboard_project_link') || ($tab.prop('id') == 'dashboard_rewards_link') || ($tab.prop('id') == 'dashboard_subgoals_link')) {
+      if(($tab.prop('id') == 'project_metrics_link') || ($tab.prop('id') == 'project_reports_link') || ($tab.prop('id') == 'basics_link') || ($tab.prop('id') == 'dashboard_project_link') || ($tab.prop('id') == 'dashboard_rewards_link') || ($tab.prop('id') == 'dashboard_subgoals_link') || ($tab.prop('id') == 'project_bank_info_link')) {
         $('#project-sidebar').hide();
       } else {
         $('#project-sidebar').show();
```

Comment:
o toggle não dá certo nesse caso, é o seguinte:
quando for um desses links `#project-sidebar` tem que estar `hide`, se não `show` 
se usar toggle, toda vez que que for um desses links ele vai mudar de `hide` pra `show` e vice versa, então as vezes o `#project-sidebar` acaba aparecendo quando não deveria